File: 백준/Bronze I/10798. 세로읽기/세로읽기.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));

        char[][] cArray = new char[5][16];
        for (int i = 0; i < 5; i++) {
            cArray[i] = reader.readLine().toCharArray();
        }

        for (int k = 0; k < 15; k++) {
            for (int i = 0; i < 5; i++) {
                if (cArray[i].length > k) {
                    System.out.print(cArray[i][k]);
                }
            }
        }

        reader.close();
    }
}

File: 백준/Bronze I/10798. 세로읽기/세로읽기.cc
#include <iostream>
using namespace std;

char cArray[5][16];  

int main(){
  for(int i=0; i<5; i++) 
    cin>>cArray[i];
  
  for(int k=0; k<15; k++)  
  {
    for(int i=0; i<5; i++)  
    {
      if(cArray[i][k] != '\0')  
        cout << cArray[i][k];
    }
  }
}